### PR TITLE
VUE-Insights-dashboard

### PR DESCRIPTION
**Pull request by j-jmnz**
# Issue:  [SC-2352](https://ticketsystem.schul-cloud.org/browse/SC-2352)

## Description

- Dashboard page view for Insights

### CODE REVIEW

**Comment by janrenz** [comment]
How large is the echarts lib? If its large, can we lazy load is only if needed?

**Comment by j-jmnz** [comment]
> How large is the echarts lib? If its large, can we lazy load is only if needed?

The size of the library is 795.3 kB, let me know if you'd prefer to lazy load the modules.

**Comment by j-jmnz** [comment]
> How large is the echarts lib? If its large, can we lazy load is only if needed?

@janrenz  The library size is 795.3 kB, please let me know you'd prefer to lazy load the modules

**Comment by adrianjost** [comment]
0.75MB should definitely be lazy loaded.

**Comment by janrenz** [comment]
Will merge and optimize from there.

File: package.json [hunk 1]
```diff
@@ -45,17 +45,20 @@
...
 		"dayjs": "^1.8.14",
+		"echarts": "^4.4.0",
 		"font-awesome": "^4.7.0",
 		"git-rev-sync": "^1.12.0",
 		"glob": "^7.1.4",
 		"jwt-decode": "^2.2.0",
 		"kjua": "^0.6.0",
 		"lodash": "^4.17.11",
 		"material-icons-svg": "^1.0.0",
+		"moment": "^2.24.0",
```
[the remark this answers is not in the document]
Comment (dominikbruechner) [reply package.json]: just plainly replaced it, needs to be tested for functionality

**Pull request merged**